Sources/_MatchingEngine/Regex/Printing/DumpAST.swift

```diff
@@ -282,7 +290,15 @@ extension AST.Quantification: _ASTPrintable {
 
 extension AST.CustomCharacterClass: _ASTNode {
   public var _dumpBase: String {
-    "customCharacterClass(\(members))"
+    // Exclude trivia for now, as we don't want it to appear when performing
+    // comparisons of dumped output in tests.
+    // TODO: We should eventually have some way of filtering out trivia for
+    // tests, so that it can appear in regular dumps.
```

Comment:
Even more so, we may want a tree transform to strip trivia, to convert trivia into semantic whitespace, to convert semantic whitespace into trivia, etc. Not super critical to add these transforms, but I think it gives us some flexibility and demonstrates that we can add conversion features